src/components/mobile-menu-container/mobile-menu-container.module.css

```diff
@@ -18,7 +18,7 @@
 	position: fixed;
 	top: var(--sticky-bars-height);
 	width: 100%;
-	z-index: 1;
+	z-index: 2;
```

Comment:
🗒️ This was needed to place the mobile menu container on top of bookmark buttons (now `z-index: 1`).